addons/product_images/wizard/product_get_pic_using_barcode.py

```diff
@@ -46,11 +46,13 @@ def default_get(self, fields):
             raise UserError(_("There isn't any product without a picture and with a barcode."))
         # TODO: Define a maximum number of product to process in the same batch
         if len(product_ids) > 100:
-            raise UserError(_("You selected too many product variants at the same time. Please "
-                              "select a maximum of 100 product variants in the same batch."))
+            raise Warning(_("You selected too many product variants at the same time. Please "
+                            "select a maximum of 100 product variants in the same batch."))
+            product_ids = product_ids[:100]
```

Comment:
Sadly that won't work, a `raise` is a `raise`, everything following it is dead code, but nice try :D

Reply:
I know :') I just wanted to inform the user that only the 100 first product variants will be processed but I didn't find yet how to do that and I forgot to remove those lines in my commit. Besides, I removed them in the next one [5f24e5d](https://github.com/odoo/odoo/pull/70198/commits/5f24e5d00a8bffb8a19628695b557163fcaaf327) ;)